config/environments/development.rb

```diff
@@ -32,9 +32,23 @@
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-
+  config.action_mailer.raise_delivery_errors = true
   config.action_mailer.perform_caching = false
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.default_options = {
+    from: ENV["MAIL_USERNAME"],
+    host: ENV['API_URL'],
+    protocol: 'http'
+  }
+  config.action_mailer.smtp_settings = {
```

Comment:
I'd suggest using something like [letter_opener](https://github.com/ryanb/letter_opener) in development, if possible.